### PR TITLE
adding code owners

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Automatically request reviews when a pull request changes any owned files
+# More information: https://github.com/blog/2392-introducing-code-owners 
+
+*.groovy @dotnet/roslyn-infrastructure
+.github/ @dotnet/roslyn-infrastructure
+build/ @dotnet/roslyn-infrastructure
+samples/ @jmarolf
```
Comment (jasonmalinowski) [inline .github/CODEOWNERS]: Should we have a team for this, especially given it's flu season? :smile:

Comment (jmarolf) [reply .github/CODEOWNERS]: We can. You want to be on it 😃

Comment (jmarolf) [reply .github/CODEOWNERS]: @jasonmalinowski I don't have permissions to create a group.  Who do I talk to about that?

Comment (jasonmalinowski) [reply .github/CODEOWNERS]: Does https://github.com/orgs/dotnet/new-team not work? I thought everybody had permissions.

Comment (jmarolf) [reply .github/CODEOWNERS]: that worked for me! I had different, old, instructions

Comment (jasonmalinowski) [reply .github/CODEOWNERS]: @jmarolf I guess I was imagining an SDK team that owns both samples and templates.

Comment (jmarolf) [reply .github/CODEOWNERS]: Sure, that can work. Templates has been only me for a while.  Don't know who would review that.

Comment (jasonmalinowski) [reply .github/CODEOWNERS]: The team can be you for now. :-)

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Automatically request reviews when a pull request changes any owned files
+# More information: https://github.com/blog/2392-introducing-code-owners 
+
+*.groovy @dotnet/roslyn-infrastructure
+.github/ @dotnet/roslyn-infrastructure
+build/ @dotnet/roslyn-infrastructure
+samples/ @dotnet/analyzer-samples
```
Comment (sharwell) [inline .github/CODEOWNERS]: :memo: Read permission for this repository needs to be added to the @dotnet/analyzer-samples team

**Pull request merged**